courses/cunix/ex01/src/my_strlen.c

```diff
@@ -0,0 +1,9 @@
+unsigned int my_strlen(char *str){
```

Comment:
```suggestion
unsigned int my_strlen(char *str) {
```
it's a good practice to put a space before '{'